src/ophyd_async/epics/signal/signal.py

```diff
@@ -58,6 +58,28 @@ def epics_signal_rw(
     return SignalRW(backend)
 
 
+def epics_signal_rw_rbv(
+    datatype: Type[T],
+    read_pv: str,
+    write_pv: Optional[str] = None,
+    read_suffix: str = "_RBV",
```

Comment:
We never need to provide a separate read pv, so suggest we change signature to:
```suggestion
    write_pv: str,
    read_suffix: str = "_RBV",
```

Reply:
Done